client/src/pages/Profile/ProfileEdit/ProfileEditForm/ProfileEditForm.tsx

```diff
@@ -128,6 +130,7 @@ export default function ProfileEditForm(): JSX.Element {
                   <Grid item className={`${classes.textboxContainer}`} justify="flex-end" xs={textboxSize}>
                     <TextField
                       className={`${classes.textbox}`}
+                      required={true}
```

Comment:
```suggestion
                      required
```
You can simply pass a boolean prop to imply truthiness.

Reply:
Thank you!